src/css.js

```diff
@@ -113,6 +113,14 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( window.top !== window.self && document.msFullscreenElement &&
+		elem.getBoundingClientRect().width > elem.offsetWidth ) {
```

Comment:
BTW, I'll be working soon on fixing #1724 so there will be more `getBoundingClientRect` around. It would be cool if it was correct in IE11 because this whole workaround wouldn't be needed then. ;)